src/StorageSync/StorageSync.Test/ScenarioTests/Common.ps1

```diff
@@ -200,6 +200,15 @@ function Get-StorageSyncLocation($provider)
     return $defaultLocation
 }
 
+<#
+.SYNOPSIS
+Gets the default location for a resource group
+#>
+function Get-ResourceGroupLocation()
+{
+	return "West US"
```

Comment:
Please use Get-Location to ensure this works on multiple clouds: https://github.com/Azure/azure-powershell/blob/master/tools/ScenarioTest.ResourceManager/Common.ps1#L583

Reply:
Switched to Get-Location. BTW, it looks like it's internally using Get-AzureRmResourceProvider, which appears to be out of date now (should be Az?)